modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,6 +1,236 @@
+
+|===
+
+=== Query Service
+
+[#table-known-issues-712-queryservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52254[MB-52254^]
+| Improve pagination queries with fetch
+
+| https://issues.couchbase.com/browse/MB-52764[MB-52764^]
+| Race condition between stop signal and timeout
+
+| https://issues.couchbase.com/browse/MB-52253[MB-52253^]
+| Push filters to index scan to potentially reduce fetch size
+
+| https://issues.couchbase.com/browse/MB-52959[MB-52959^]
+| UNNEST query 'usedMemory' issue when using Query Memory Quota
+
+| https://issues.couchbase.com/browse/MB-52443[MB-52443^]
+| Include missing key Index syntax on Index workbench doesn't show include keyword in definition
+
+| https://issues.couchbase.com/browse/MB-31640[MB-31640^]
+| subqueries should be advised, explained and monitored
+
+|===
+
+=== Index Service
+
+[#table-known-issues-712-indexservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52063[MB-52063^]
+| With Collection Indexer should index leading MISSING entries
+
+| https://issues.couchbase.com/browse/MB-51947[MB-51947^]
+| indexer blocked during storage warmup on MOI storage, causing rebalance failure.
+
+| https://issues.couchbase.com/browse/MB-52443[MB-52443^]
+| Include missing key Index syntax on Index workbench doesn't show include keyword in definition
+
+|===
+
+=== Search Service
+
+[#table-known-issues-712-searchservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-51760[MB-51760^]
+| GeoJSON Spatial Indexing support
+
+|===
+
+=== Analytics Service
+
+[#table-known-issues-712-analyticsservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52783[MB-52783^]
+| Select * query throws Failure contacting server for parquet files
+
+|===
+
+
+=== Eventing Service
+
+[#table-known-issues-712-eventingservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52376[MB-52376^]
+| CI test for eventing multi collection
+
+| https://issues.couchbase.com/browse/MB-52808[MB-52808^]
+| Eventing function deployment taking a long time
+
+| https://issues.couchbase.com/browse/MB-52365[MB-52365^]
+| appcode rest api returns bytes instead of string
+
+| https://issues.couchbase.com/browse/MB-52367[MB-52367^]
+| Log function scope for lifecycle operation audit logs
+
+| https://issues.couchbase.com/browse/MB-52372[MB-52372^]
+| Unable to modify function settings when user has only eventing_manage_functions role
+
+| https://issues.couchbase.com/browse/MB-52587[MB-52587^]
+| Eventing Multi Collection: Inter function recursion not detected in case of sbm handler
+
+| https://issues.couchbase.com/browse/MB-52369[MB-52369^]
+| Eventing Service should honor the CPU & Memory limits set in cgroups
+
+| https://issues.couchbase.com/browse/MB-52373[MB-52373^]
+| Narrow UI's sometimes drop the Eventing service from the alternate menu
```

Comment:
@tonyjhillman This is not a BUG (UI people claim it was a feature) - no change was done so customers do not need to see this MB

Reply:
Removed.